core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java

```diff
@@ -93,7 +94,7 @@ public void handle(final RoutingContext routingContext) {
                 }
               },
               () -> {
-                LOG.trace("Identifier not found {}", identifier);
+                LOG.warn("Identifier not found {}", identifier);
```

Comment:
Don't think we need this log at all as when no signer was found we log an error in DefaultArtifactSignerProvider, log at warn here and then finally log in the LogErrorHandler so all up we get 3 logs entries.

e.g.
```
2020-10-06 15:22:55.650+10:00 | vert.x-worker-thread-5 | ERROR | DefaultArtifactSignerProvider | No signer was loaded matching identifier '0x900884a719c418272c1c2b038f3dd5629a4bf2465d30963d181f454565f4236c3441aa49049c9fedac57c381fb067a2a'
2020-10-06 15:22:55.650+10:00 | vert.x-worker-thread-5 | WARN  | Eth2SignForIdentifierHandler | Identifier not found 0x900884a719c418272c1c2b038f3dd5629a4bf2465d30963d181f454565f4236c3441aa49049c9fedac57c381fb067a2a
2020-10-06 15:22:55.650+10:00 | vert.x-eventloop-thread-4 | ERROR | LogErrorHandler | Failed request: http://localhost:9000/api/v1/eth2/sign/0x900884a719c418272c1c2b038f3dd5629a4bf2465d30963d181f454565f4236c3441aa49049c9fedac57c381fb067a2a
```

Reply:
👍

Reply:
done